src/components/KakaoMaps/StaticKakaoMap.tsx

```
@@ -1,44 +1,72 @@
-                },
-            ]}
-            level={l}
-        />
-    );
+  const [isKakaoLoaded, setIsKakaoLoaded] = useState(false);
+
+  useEffect(() => {
+    const checkKakao = () => {
+      if (window.kakao && window.kakao.maps) {
+        setIsKakaoLoaded(true);
+      } else {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)